DESCRIPTION

```diff
@@ -64,18 +64,18 @@ Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.2.0
 Depends: R (>= 3.5)
 Imports:
-    assertthat,
-    dplyr,
-    hms,
-    lifecycle,
-    lubridate,
-    magrittr,
-    purrr,
-    rlang,
-    stringr,
-    tidyr,
-    tidyselect,
-    readxl
+    assertthat (>= 0.2.1),
+    dplyr (>= 1.0.0),
```

Comment:
I'm curious, do we actaully use any feature that got introduced in version 1.0.0?